jetson/odrive_bridge/src/__main__.py

```diff
@@ -515,28 +393,15 @@ def _requested_state(self, state):
         self.front_axis.requested_state = state
 
     def set_vel(self, axis, vel):
-        global legal_controller
         if (axis == "LEFT"):
-            # TEMPORARY FIX FOR ROLLING ROVER SINCE
-            # middle left odrive IS 2x more than the rest bc of the 48V maxon
-            # TODO - fix when this is no longer the case!
-            if (legal_controller == 1):
-                self.front_axis.controller.input_vel = vel * 100
-            else:
-                self.front_axis.controller.input_vel = vel * 50
+            self.front_axis.controller.input_vel = -vel * 3

```

Comment:
Out of curiosity, what is this code doing, specifically with the 3?

Reply:
lol that looks like smth wrong happened during the rebase, good catch